core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -65,23 +65,23 @@
       In addition, if the exploration is iframed, the terminal card will
       have no learner input section, so we do not show it.
     -->
-    <div ng-show="isInteractionInline && isCurrentCardAtEndOfTranscript() && !waitingForOppiaFeedback &&
+    <div ng-show="isInteractionInline && isCurrentCardAtEndOfTranscript() && !waitingForOppiaFeedback && !isConceptCardPending() &&
```

Comment:
Why is the additional check needed here? I can understand a check based on "is being shown", but I'm not sure I understand the pending part.

Reply:
This was because, the interaction was hidden only when a correct answer was given (in which case destStateName is not null), and for answers that stayed on the same card, the interaction would be displayed, but when a concept card is pending, the feedback needs to be displayed without an interaction below it. Hence, the check.

Reply:
Hm OK. I think that's not apparent from reading this. Can we pull the logic into a function with appropriate comments or intermediate var names that makes it really clear what's going on?

Reply:
This needs to be removed now based on my comment on invalid answer issue.

Reply:
Done, removed this.